### ISSUE TITLE
falco Readiness probe fails

### ISSUE THREAD

**Issue by tsigian** [PacktPublishing/Kubernetes---An-Enterprise-Guide-2E#3]
host: Ubuntu 20.04 Focal Fossa LTS
Chapter 10

Got ebpf support, I was able to run `'sudo opensnoop-bpfcc'`

Not sure why the readiness probe keeps failing, maybe you have some suggestions ?

`$ k describe pod falco-t5qh2`
`Warning  Unhealthy  12s (x2 over 27s)  kubelet            Readiness probe failed: Get "http://172.18.0.3:8765/healthz": dial tcp 172.18.0.3:8765: connect: connection refused
`

Logs from the kubelet :

`$ docker exec -it 5b5b425cabc7 bash`
`root@cluster01-worker:/# journalctl -u kubelet -f`

`Feb 18 20:32:24 cluster01-worker kubelet[242]: I0218 20:32:24.180431     242 scope.go:111] "RemoveContainer" containerID="5113d12dffa8f560bec9779371a552415f13e5b9419c8e1ae6d5f08a0630f3bc"
Feb 18 20:32:24 cluster01-worker kubelet[242]: E0218 20:32:24.181735     242 pod_workers.go:190] "Error syncing pod, skipping" err="failed to \"StartContainer\" for \"falco\" with CrashLoopBackOff: \"back-off 2m40s restarting failed container=falco pod=falco-442ng_falco(c4db0dd4-d751-40a4-8f6d-cc19164d8915)\"" pod="falco/falco-442ng" podUID=c4db0dd4-d751-40a4-8f6d-cc19164d8915`

**Comment by ssurovich** [comment]
Hi @tsigian - It looks like there are a few reported issues with Falco and the building, seems to be in kernel 5.4 most often - but I reproduced your issue and I tested the newest Falco after the failure and it deployed without any issues.

I would delete your Falco using helm delete falco -n falco, and then pull the updated repo (or edit your values-falco.yaml file and change the version from 0.29.1 to 0.31.0) and run ./install-falco.sh again.   

Ill keep this open until you test the 0.31.0 release.
